jenkinsapi_tests/systests/test_quiet_down.py

```diff
@@ -1,11 +1,28 @@
+
 """
 System tests for setting jenkins in quietDown mode
 """
+import pytest
+import logging
+
 
+log = logging.getLogger(__name__)
 
-def test_quiet_down(jenkins):
+
+@pytest.mark.run_these_please
+def test_quiet_down_and_cancel_quiet_down(jenkins):
     jenkins.poll()  # jenkins should be alive
     jenkins.quiet_down()  # put Jenkins in quietDown mode
+
+    jenkins_api = jenkins.get_api_python_data()
+    is_quieting_down = jenkins_api['quietingDown']
```

Comment:
This code is not part of test. There shall be a property or method in `Jenkins` object that returns `quietingDown` value.

Reply:
Would it be possible to implement getting the 'api/python' in jenkinsbase.py, parsing the content to a python object, and return the python object. This would make it a one-time endeavour, making it compatible with any content in this object?